provider/bluecat/bluecat_test.go

```diff
@@ -227,7 +241,9 @@ func TestBluecatRecords(t *testing.T) {
 	for _, ti := range tests {
 		actual, err := provider.Records(context.Background())
 		if err != nil {
-			t.Fatal(err)
+			if !strings.Contains(err.Error(), "External-DNS Owner not found") {
+				t.Fatal(err)
+			}
```

Comment:
Since we are no longer returning an error when the external dns owner is not found, I think we could leave this block as it was originally.